packages/ui/src/theme/tokens/components/select.ts

```diff
@@ -59,12 +70,18 @@ export const select: Required<SelectTokens<'default'>> = {
       right: { value: '{space.medium.value}' },
     },
   },
-  // for Firefox only, to fix background color in darkmode
+  // It's important to test these option values on
+  // Chrome/FireFox/Edge on Windows and Firefox on
+  // Mac because they allow styling of the option element.
+  // Chrome/Safari on Mac uses the system ui.
```

Comment:
```suggestion
  // It's important to test these option values on Chrome/FireFox/Edge 
  // on Windows because they allow styling of the option element. 
  // Chrome/Safari/Firefox on Mac uses the system ui.
```